source/index.html.md

```diff
@@ -206,10 +206,33 @@ This endpoint loads customers by csv files
 | 404  | Invalid businessId or no csv file supplied                                        |
 
 
+## Obtain the results of a customer csv import via import id
 
+This endpoint fetches a particular result of loading a CSV file
 
-# Subscriptions
+### HTTP Request
+
+`GET` /enterprise/v1/customers/{businessId}/upload/{importId}
+
+### Path Parameters
+
+| Parameter            | Description                                                    |
+|----------------------|----------------------------------------------------------------|
+| `businessId`         | The unique account key of the business                         |
+| *string*, *path*     |                                                                |
+| `importId`           | The unique number used to identify the record in the database  |
+| *string*, *path*     |                                                                |
 
+### Responses
+
+| Code | Description                                                                                                             |
+|------|----------------------------------------------------------------------------------|
+| 200  | A report object showing which of the customer records were successfully processed| 
+| 403  | Operation not permitted for this business                                        |
+| 404  | Invalid businessId or importId supplied                                          |
+
+
```

Comment:
Where is the example json output? Run the `enterprise-openapi`. The example format is there now.